config/database.yml

```diff
@@ -0,0 +1,42 @@
+base: &default
+<% case ENV['DATABASE_URL'].to_s
```

Comment:
So, this might be controversial... 
This changed file does the `ENV['DATABASE_URL'].to_s` 4 times :grimacing:  (well, I guess at the execution time it will be 2.
But to be honest, for me it's much easier to understand this one than the original one: https://github.com/3scale/porta/blob/539780953ebc6eddebcc1450a842e9d3198ea571/config/examples/database.yml

But maybe I'm wrong, and the original is actually better. Opinions welcome!

Reply:
I changed my mind :upside_down_face: 

I will revert the format of this file to the original one.